infrastructure/tfvars_tests/standard-gke-public.platform.tfvars

```diff
@@ -50,10 +50,9 @@ cpu_pools = [{
 ## make sure required gpu quotas are available in the corresponding region
 enable_gpu = true
 gpu_pools = [{
-  name           = "gpu-pool-l4"
-  machine_type   = "g2-standard-24"
-  node_locations = "us-central1-a"
```

Comment:
I think the reason we had node locations was so that we don't spawn a replica in all zones. We should probably leave in a single zone for us-east4 too, to avoid stockouts

Reply:
updated to use us-east4-c